Jint/Runtime/Interop/DefaultTypeConverter.cs

```diff
@@ -1,8 +1,12 @@
 ﻿using System;
 using System.Collections.Concurrent;
+using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Dynamic;
+using System.Linq;
```

Comment:
I dislike adding LINQ as it's always slower than using types directly, but could be tweaked later on, seems that there are no captured closures though which is good